### PR TITLE
Added Firebase Hosting

### PR DESCRIPTION
**Pull request by timzprof**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+  "name": "dsc-unilag.github.io",
+  "version": "1.0.0",
+  "description": "<p align=\"center\">\r <img width=\"300\" height=\"140\"  src=\"https://github.com/kelvinkamau/Vibranium/blob/master/images/vibranium.png\">\r </p>",
```
Comment (ashinzekene) [inline package.json]: Does it accept html?

File: package.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  "main": "service-worker.js",
+  "scripts": {
+    "serve": "firebase serve"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/timzprof/dsc-unilag.github.io.git"
```
Comment (ashinzekene) [inline package.json]: Can you change this url to point to our github?

File: package.json (hunk 1 above)
Comment (timzprof) [reply package.json]: Yes it does

File: package.json (hunk 2 above)
Comment (timzprof) [reply package.json]: Yeah I can, I forgot to check the comments before merging , I reverted it
I actually can't do anything cause I wont have access to my laptop till sunday.

Comment (ashinzekene) [reply package.json]: OkY

**Pull request merged**